examples/huggingface/pytorch/text-generation/quantization/run_generation.py

```diff
@@ -17,6 +17,9 @@
 import numpy as np
 from itertools import chain
 from optimum.utils import NormalizedConfigManager
+# ipex dependency

```

Comment:
do you need this change? assuming these would be hided into llm API?

Reply:
The SmoothQuantConfig backend is ipex only, so it is necessary import IPEX, yes, I can hide it to llm API.